src/main/resources/org/jenkinsci/plugins/workflow/job/properties/DurabilityHintJobProperty/help.html

```diff
@@ -22,7 +22,29 @@
  ~ THE SOFTWARE.
  -->
 <div>
-    Durability hints that guide how we trade off between performance and durability against failures.
+    <p>
+        This setting allows users to change the default durability mode for running Pipelines.  In most cases this is a
+        trade-off between performance and the ability for running pipelines to resume after unplanned Jenkins outages.
 
-    <p>This may also be set globally under Manage Jenkins > Configure System.
+        <br><br><strong>What does this do?</strong>
```

Comment:
Does this not duplicate help text in `workflow-api`? That is not going to be maintainable.

If you need to write up an elaborate explanation, it is perhaps better to define something like https://jenkins.io/redirect/pipeline-durability/ which could be a wiki page, article, whatever (can be changed at any time), and let each point in the UI which touches on this link to it for all details.